layouts/blog/rss.xml

```diff
@@ -0,0 +1,29 @@
+<rss version="2.0">
+    <channel>
+        <title>Pulumi Blog</title>
+        <link>{{ .Permalink }}</link>
+        <description>Infrastructure as Code for AWS, Azure, Google Cloud, including Kubernetes, Serverless, Containers, for Developers and DevOps.</description>
+        <language>{{ .Site.LanguageCode }}</language>
+        <pubDate>{{ .Date.Format "Mon, 02 Jan 2006" | safeHTML }}</pubDate>
```

Comment:
According to the [RSS 2.0 Spec](https://cyber.harvard.edu/rss/rss.html), these should conform to the Date and Time Specification of [RFC 822](https://www.ietf.org/rfc/rfc822.txt), which I believe should include the time and timezone.

From https://gohugo.io/templates/rss/:

```
<pubDate>{{ .Date.Format "Mon, 02 Jan 2006 15:04:05 -0700" | safeHTML }}</pubDate>
```